### ISSUE TITLE
Bug: Cannot read property 'eq' of null

### ISSUE THREAD

**Issue by jloosli** [fullcalendar/fullcalendar#3918]
## What is happening
When using Fullcalendar Scheduler with `refetchOnNavigate` and `nowIndicator` and a resource callback, if you switch in and out of the agendaDay view, you get the following error:

```
errors.js:48 ERROR TypeError: Cannot read property 'eq' of null
    at members.constructor.renderNowIndicator (fullcalendar.js:17222)
```

## What should happen
You should be able to switch back and forth as many times as you want

## To replicate
* Go to https://stackblitz.com/edit/angular-fullcalendar-issue
* Switch to Day view
* Switch to Week or Month View
* Switch to Day view
* You'll see the error in the console

## Misc
This happens to be in an Angular project, but it doesn't look like Angular interferes in this case

**Comment by arshaw** [comment]
hi @jloosli any way you can get this error to happen without angular?

**Comment by jloosli** [comment]
Here you go: 
* Go to https://codepen.io/anon/pen/rYWXOv?editors=0010
* Click "day"
* Click "today"
* Click "week"
* Click "day"

You should see the error in your console.

**Comment by arshaw** [comment]
confirmed, thanks

**Comment by arshaw** [comment]
Released in core [v3.7.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.7.0) and scheduler [v1.9.0](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.9.0).
